src/main/webapp/ts/app.component.ts

```diff
@@ -331,7 +331,7 @@ export class AppComponent {
             }
           } else {
             if (data1.output && data1.output.text) {
-              responseText = data1.output.text.length === 2 && !data1.output.text[0] ? data1.output.text[1] : data1.output.text[0];
+              responseText = data1.output.text.length >= 1 && !data1.output.text[0] ? data1.output.text.join(' ').trim() : data1.output.text[0]; // tslint:disable-line max-line-length
```

Comment:
Had to disable tslint for checking max line as this line isn't such complex; we are checking if the array of text is more than one and are joining the array's and trimming the text if not we are setting the text to the first array.